Pollo/ViewControllers/PollsViewController.swift

```diff
@@ -406,22 +439,36 @@ class PollsViewController: UIViewController {
             self.joinSessionButton.alpha = canJoin ? 1.0 : 0.5
         }
     }
+
+    func updateCreateSessionButton(canCreate: Bool) {
+        UIView.animate(withDuration: joinSessionButtonAnimationDuration) {
+            self.createSessionButton.backgroundColor = canCreate ? .polloGreen : .blueGrey
+            self.createSessionButton.alpha = canCreate ? 1.0 : 0.5
+        }
+    }
     
     @objc func settingsAction() {
         let settingsNavigationController = UINavigationController(rootViewController: SettingsViewController())
         self.present(settingsNavigationController, animated: true, completion: nil)
     }
     
-    @objc func didStartTyping(_ textField: UITextField) {
+    @objc func didStartTypingCode(_ textField: UITextField) {
         if let text = textField.text {
             textField.text = text.uppercased()
             updateJoinSessionButton(canJoin: text.count == IntegerConstants.validCodeLength)
         }
     }
+
+    @objc func didStartTypingName(_ textField: UITextField) {
```

Comment:
nit: can you rename this to something more descriptive? Like `didStartTypingGroupName`?